CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 ## Latest
-* Update Roslyn version to 4.8.0-3.23472.2 and Razor version to 8.0.0-preview.23465.2 (PR: )
+* Update Roslyn version to 4.8.0-3.23472.2 and Razor version to 8.0.0-preview.23465.2 (PR: [#6423](https://github.com/dotnet/vscode-csharp/pull/6423))
```

Comment:
this needs to be in a new changelog section since we already released everything below this

Could you create a new latest section with this change listed, and put a new heading for `2.3.27` below?

Reply:
Done!